tests/integration/Core/Checkout/Document/Service/DocumentConfigLoaderTest.php

```diff
@@ -22,13 +25,16 @@
 class DocumentConfigLoaderTest extends TestCase
 {
     use DocumentTrait;
+    use IntegrationTestBehaviour;
```

Comment:
please do not use this overkill trait of traits. use only the ones you really need. like the KernelTestBehaviour or the database trait